spring-xd-module/src/main/java/org/springframework/xd/module/options/DefaultModuleOptionsMetadataResolver.java

```diff
@@ -101,15 +101,7 @@
 
 	private ResourcePatternResolver resourceLoader = new PathMatchingResourcePatternResolver();
 
-	/**
-	 * Construct a new {@link DefaultModuleOptionsMetadataResolver} that will use the provided conversion service when
-	 * converting from String to rich object (supported for {@link PojoModuleOptionsMetadata} only).
-	 *
-	 * @param conversionService Must not be null
-	 */
-	public DefaultModuleOptionsMetadataResolver(ConversionService conversionService) {
-		Assert.notNull(conversionService, "The provided conversionService must not be null.");
-	}
+	private boolean useParentClassLoader = false;
```

Comment:
This is slightly confusing given that the classloader only becomes a "parent" IF a child classloader is created. In other words, when this boolean is set to TRUE, then there actually is no parent classloader. Perhaps it would be clearer to flip the boolean (so it's true by default and would be set to false by the new modules) and use a name for the variable that indicates a new child classloader would be created if it's true. Does that make sense?